vsetup.sh

```diff
@@ -104,15 +104,23 @@ function env_all_pre_centos () {
 }
 function env_all_pre_ubuntu () {
     export DEBIAN_FRONTEND=noninteractive
-    say "Running apt-get update -qq ..."
-    apt-get update -qq
-    apt-get install software-properties-common
+    say "Running apt-get update ..."
+    apt-get update -y -qq
+    apt-get install -y software-properties-common apt-transport-https
+
+    say "Adding Zimbra repository ..."
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 9BE6ED79
+    cat > /etc/apt/sources.list.d/zimbra.list <<EOF
+deb [arch=amd64] https://repo.zimbra.com/apt/87 $(lsb_release -cs) zimbra
```

Comment:
Adding the zimbra repo may not be desirable for folks who want to build 3rd party out themselves.  I'd make it an option.